x/emissions/keeper/topic_activation.go

```diff
@@ -41,15 +41,21 @@ func (k *Keeper) GetActiveTopicIdsAtBlock(ctx context.Context, block BlockHeight
 }
 
 // Boolean is true if the block is not found (true if no prior value), else false
-func (k *Keeper) GetLowestActiveTopicWeightAtBlock(ctx context.Context, block BlockHeight) (types.TopicIdWeightPair, bool, error) {
-	weight, err := k.blockToLowestActiveTopicWeight.Get(ctx, block)
+func (k *Keeper) GetLowestActiveTopicWeightAtBlock(
+	ctx context.Context,
+	block BlockHeight,
+) (topicIdAndWeight types.TopicIdWeightPair, noPrior bool, err error) {
+	topicIdAndWeight, err = k.blockToLowestActiveTopicWeight.Get(ctx, block)
 	if err != nil {
 		if errors.IsOf(err, collections.ErrNotFound) {
-			return types.TopicIdWeightPair{}, true, nil
+			return types.TopicIdWeightPair{
+				TopicId: 0,
```

Comment:
Consider dedicated `ZERO_TOPIC_ID` constant, but nbd

Reply:
there's only two places this occurs, in this file and in quantile_test.go, don't think a constant is warranted